zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/CollectionHistory.java

```diff
@@ -60,9 +60,7 @@ private void checkPermission(Session session) throws ZebedeeException {
                 throw new UnauthorizedException("You are not authorised to create collections.");
             }
         } catch (IOException io) {
-            logError(io, "Unexpected error while trying to access permissions")
-                    .user(session.getEmail())
-                    .logAndThrow(UnexpectedErrorException.class);
+            error().data("user", session).logAndThrow (io, "Unexpected error while trying to access permissions");
```

Comment:
`session.getEmail()`

Reply:
I think this needs to have `throw` in front - the new log n throw method doesn't throw an exception is returns an exception so you have to declare a throw in order for it to happen... possibly a better name for this method is needed.

```
throw error().data("user", session).logAndThrow (io, "Unexpected error while trying to access permissions");
```

Reply:
probably ok as you had it, this is more a bad assumption on my part that a code issue.